apps/ledger-live-mobile/ios/ledgerlivemobile/AppDelegate.m

```diff
@@ -97,6 +98,7 @@ - (void) initializeFlipper:(UIApplication *)application {
     [client addPlugin: [[FKUserDefaultsPlugin alloc] initWithSuiteName:nil]];
     [client addPlugin: [FlipperKitReactPlugin new]];
     [client addPlugin: [[FlipperKitNetworkPlugin alloc] initWithNetworkAdapter:[SKIOSNetworkAdapter new]]];
+    [client addPlugin: [FlipperPerformancePlugin new]];
```

Comment:
If I'm not wrong we're in 0.68.5 and the doc says to implement this part differently (https://github.com/bamlab/react-native-flipper-performance-monitor#-v0680)

Reply:
Thanks @Justkant, I did, it's just that some lines existed before (ifdef, FlipperClient) so I added only the missing ones.